.mvn/maven.config

```diff
@@ -1 +1 @@
--DaltSnapshotDeploymentRepository=repo.spring.io::default::https://repo.spring.io/libs-snapshot-local -s settings.xml -P spring
+-DaltSnapshotDeploymentRepository=repo.spring.io::default::https://repo.spring.io/libs-snapshot-local -P spring
```

Comment:
But don't we still need `settings.xml`? We use that in all dataflow maven wrappers.